examples/minimal/packages/contracts/mud.config.mts

```diff
@@ -1,4 +1,4 @@
-import { mudConfig, resolveTableId } from "@latticexyz/config";
+import { mudConfig, resolveTableId } from "@latticexyz/world/register"
```

Comment:
technically any endpoint works, the point of `world/library` is to more explicitly separate stuff without side-effects from stuff with side-effects. And `world/register` is named thus mostly because of hardhat's conventions, config works as well I suppose, just isn't as clear to me that it has side-effects

Reply:
got it - it wasn't immediately clear to me why it's called `register`, but if this is a convention from hardhat I think it's fine